third_party/CudaArchitectureSelector/CMakeLists.txt

```
@@ -1,6 +1,6 @@
 ginkgo_load_git_package(CudaArchitectureSelector
     "https://github.com/ginkgo-project/CudaArchitectureSelector.git"
-    "0b46fb7d653404db312cbc1fc702cb528fd1c1b0")
+    "a2d58da061c9f9ec30f20ba9ecf9959b78772c30")
```

```suggestion
    "51e6fc3590bfadded3bcf56f739c90066b2b8071")
```
use the commit in the master branch